opencost-ui.yaml

```diff
@@ -0,0 +1,82 @@
+package:
+  name: opencost-ui
+  version: 1.114.0
+  epoch: 0
+  dependencies:
+    runtime:
+      - nginx
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - nodejs
+      - npm
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: dcbd21f018fa991ddbab8a71e356b9d74f115c31
+      repository: https://github.com/opencost/opencost-ui
+      tag: v${{package.version}}
+
+  - name: build
+    runs: |
+      npm install
+      npx parcel build src/index.html
+      mkdir -p ${{targets.contextdir}}/opt/ui/dist
+      mkdir -p ${{targets.contextdir}}/var/www
+      mkdir -p ${{targets.contextdir}}/usr/local/bin
+      adduser -u 1001 -D nginx
+      mkdir -p ${{targets.contextdir}}/var/log/nginx
+      cp ./dist/* ${{targets.contextdir}}/opt/ui/dist
+      cp THIRD_PARTY_LICENSES.txt ${{targets.contextdir}}/THIRD_PARTY_LICENSES.txt
```

Comment:
This can be included as a license reference under the copyright section

Reply:
just like this?

```
  copyright:
    - license: Apache-2.0
    - license: THIRD_PARTY_LICENSES.txt
```

Reply:
Like this:

```
- license: LicenseRef-Third-Party
  license-path: ./THIRD_PARTY_LICENSE.txt
```